cmd/kube-controller-manager/app/controllermanager.go

```diff
@@ -241,9 +241,14 @@ func (s *CMServer) Run(_ []string) error {
 	resourceQuotaController := resourcequotacontroller.NewResourceQuotaController(kubeClient)
 	resourceQuotaController.Run(s.ResourceQuotaSyncPeriod)
 
-	namespaceController := namespacecontroller.NewNamespaceController(kubeClient, s.NamespaceSyncPeriod)
+	namespaceController := namespacecontroller.NewNamespaceController(kubeClient, s.EnableHorizontalPodAutoscaler, s.NamespaceSyncPeriod)
 	namespaceController.Run()
```

Comment:
Can we make EnableHorizontalPodAutoscaler more generic?  There may be other experimental resources that will need to be cleaned up when a namespace is deleted.